onnx_tf/handlers/backend/unique.py

```diff
@@ -0,0 +1,95 @@
+import tensorflow as tf
+import numpy as np
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+from onnx_tf.common import exception
+
+tf.config.run_functions_eagerly(True)
```

Comment:
This might not work because we need to persist the computational graph as a saved model. Please write a unit test in test_model.py. Basically we want to know for sure that a unique operator can be saved in model and executed properly from the saved model.

Reply:
https://github.com/onnx/onnx-tensorflow/issues/955
still not reply get about how is the input data likes